titus_isolate/config/property_change_handler.py

```diff
@@ -0,0 +1,24 @@
+import schedule
+
+from titus_isolate.utils import get_logger
+
+log = get_logger()
+
+
+class PropertyChangeHandler:
+    def __init__(
+            self,
+            key,
+            func,
+            property_provider,
+            change_detection_interval):
+        self.__key = key
+        self.__func = func
+        self.__property_provider = property_provider
+
+        schedule.every(change_detection_interval).seconds.do(self.__update_property)
+
+    def __update_property(self):
+        value = self.__property_provider.get(self.__key)
+        log.info("Handling property change for: '{}', to: '{}''".format(self.__key, value))
```

Comment:
maybe only `log.info` if `value is not None`?

Reply:
Dropping log level to `debug`.